test/com/facebook/buck/apple/AppleBundleIntegrationTest.java

```diff
@@ -622,6 +622,16 @@ public void appBundleWithResources() throws Exception {
             "%s"));
   }
 
+  @Test
+  public void appBundleWithConflictingResources() throws Exception {
```

Comment:
Can you add another test where we have duplicate nested filenames, so something like:

    FolderA/Image.png
    FolderB/Image.png

Reply:
How can I add a test to cover `DirsContainingResourceDirs` too?

Reply:
@rmaz As per my above comment, I don't think we can easily test this as `js_bundle` doesn't seem to have integration tests.